Parametric Query Optimization

Skialith opened the pull request:
Adding content of Parametric Query Optimization

Skialith committed:
Add Parametric Query Optimization section to README

Skialith committed:
Revise README with new PQO sections

Updated the README to include new sections on Parametric Query Optimization, including Foundational Theory, Engineering & Data-driven PQO, and ML-based PQO & Robust Query Optimization.

Skialith committed:
Fix formatting of Parametric Query Optimization section

Skialith committed:
Fix formatting of Stargazers section in README

Skialith committed:
Update references in README for query optimization

Skialith committed:
Reorganize sections in README for clarity

LumingSun commented:
Thanks for your contribution! 
I noticed some paper overlaps between your Parametric Query Optimization section and the existing Query Optimization chapter. It would be better to restructure your content as a subchapter under Query Optimization to avoid redundancy.

Skialith committed:
reconstruct Parametric Query Optimization Chapter to Query Optimization

Added detailed sections on Parametric Query Optimization, including foundational theory, engineering & data-driven PQO, and ML-based PQO.

Skialith commented:
I have reorganized the PQO chapter to Query Optimization，thank you for your correction！

Skialith committed:
Reorder Query Plan section in README
